src/main/java/org/jenkinsci/plugins/vSphereCloudSlaveTemplate.java

```diff
@@ -349,6 +352,38 @@ protected Object readResolve() {
                 LOGGER.log(Level.CONFIG, " - Failed to reconfigure strategy", ex);
             }
         }
+        // Earlier versions of the code may have stored configuration in a
+        // different way but in the same kind of fields, so we need an explicit
+        // versioning to know how to mutate the data.
+        if( configVersion<=0) {
```

Comment:
nit: inconsistent parens (looks like this project generally goes for the standard-ish `if (...) {` with space after the if and space after the close paren.)

Reply:
I'm quite happy with nit-picking comments; I'll sort those out - I dislike messy/inconsistent code too.